### DISCUSSION TITLE
6.2.2 (Grid effects beta)

### DISCUSSION THREAD

**Discussion by T-Troll** [T-Troll/alienfx-tools#156] [Announcements]
### Changes in this release:

`AlienFX-GUI`:
- New: Grid effect trigger "Event" implemented. It starts grid effect according to zone settings at "Event monitoring" tab.
- New: Grid effects configuration now saved and loaded between sessions.
- New: Alienware m15R6 (4-zone keyboard) lights and grids configuration added to device database.
- "Devices and grids" dialog keyboard shortcuts re-worked, now it dialog-only, not system.
- Haptics data save/load procedure re-worked.
- Bug fixed: Network traffic can stick at 0 at low network load.
- Bug fixed: App crashes if user press "Reset" button at "Devices and Grids" tab for undefined light.

<hr /><em>This discussion was created from the release <a href='https://github.com/T-Troll/alienfx-tools/releases/tag/6.2.2'>6.2.2 (Grid effects beta)</a>.</em>

**Comment by MarlosLucas** [comment]
Hey, T-Troll

One doubt about the version updates of AlienFX. What is the recommendation to keep the software updated? Should I always unistall and install the newer or there is another way? (perhaps extracting the zip file on the folder).

Cheers!

**Reply by T-Troll** [reply]
zip is the same as msi, so you can use it in most cases (then new app not added/removed).
Update is up to you (you can also disable update check and do it manually), decide after reading release notes.
